Sendgrid.Webhooks/Events/WebhookEventBase.cs

```diff
@@ -14,6 +14,12 @@ public WebhookEventBase()
             UniqueParameters = new Dictionary<string, string>();
         }
 
+        [JsonProperty("sg_event_id")]
+        public string SendGridEventID { get; set; }
```

Comment:
I think it makes sense for the property names to be SendgridEventId (to follow the naming conventions)